publish.gradle

```diff
@@ -93,7 +93,7 @@ publishing {
                 // Remove <packaging>pom</packaging> as it should be the default jar
                 def children = asNode().children()
                 def versionIndex = children.indexOf(children.find {it.name().localPart == 'packaging'})
-                children.remove(versionIndex)
+                if (versionIndex != -1) children.remove(versionIndex)
```

Comment:
If this isn't present anymore sounds like the fixed the bug that required this workaround. So I think this whole block can just be deleted.